packages/@lwc/template-compiler/src/codegen/index.ts

```diff
@@ -306,7 +306,11 @@ function transform(codeGen: CodeGen): t.Expression {
         return codeGen.genIterator(iterable, iterationFunction);
     }
 
-    function computeAttrValue(attr: Attribute | Property, element: BaseElement): t.Expression {
+    function computeAttrValue(
+        attr: Attribute | Property,
+        element: BaseElement,
+        addLegacySanitizationHook: Boolean
```

Comment:
Prefer primitive type over object type.

```suggestion
        addLegacySanitizationHook: boolean
```